docs/index.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Filecoin Virtual Machine"
+description: "The Filecoin Virtual Machine (FVM) allows users to write their own smart-contracts and run them against the Filecoin network. This website contains all the documetation for the FVM project, including examples and reference material to help developers build on the FVM."
+lead: "The Filecoin Virtual Machine (FVM) is an interface that developers can use to deploy smart-contracts on the Filecoin network. FVM contracts will introduce the ability to perfom on-chain computation, or the computation of a state. This is the perfect pairing of blockchain storage and computation that Web3 has been waiting for."
+---
+
+# The Filecoin Virtual Machine
+
+The Filecoin Virtual Machine (FVM) is an interface that enables developers to deploy smart contracts on the Filecoin blockchain. FVM support for smart contracts provides the ability to perform on-chain state computations that guide storage deals from pre-commit through committed states. This is the perfect pairing of blockchain storage and computation that Web3 has been waiting for.
+
+## Why the FVM is useful
+
+There has been a huge demand from the developer community for compatibility with Ethereum and Solidity out of the box. With the FVM, developers can access the massive corpus of audited and battle-tested smart contracts written in Solidity for the Ethereum Virtual Machnine (EVM). The FVM also supports other foreign runtimes and virtual machines, such as JS/SES, LLVm, IR, and BPF.
+
+The FVM is flexible. Because it's not tightly-coupled to Filecoin, developers can adapt to it in other contexts, such as the Interplanetary File System (IPFS) and Interplanetary Linked Data (IPLD).
+
+The FVM is easy for developers who know IPLD to adapt to. The state tree and actor states that the FVM uses to guide storage deals from pre-commit through committed states are IPLD objects.
+
+## Vision and goals
+
+To mitigate the well-known risks of compiling Solidity used by the EVM, we're not working directly in Solidity. Instead, we're building the reference SDK in Rust, because it produces very succinct WASM code and keeps runtime costs down.
```

Comment:
> To mitigate the well-known risks of compiling Solidity used by the EVM

What well-known risks?

Reply:
@johnnymatthews That will take research, maybe a lot. Even the professors at MIT could not articulate it well. I have some places to start looking. I understand your requests, and they're great points, but the challenge is: the reason that FVM is useful and our vision and goals are actually very technical, so I tried to keep it light and not go into detail for the c-suite.  But these are great points, so I'll try to find another approach. 
Maybe instead of pointing out the technical reasons, it would suffice to say that it is well known that Solidity has weaknesses that are resulting in the loss of millions of dollars a week. That is what I know so far. I thought one reason was that it was Turing Complete, but your guys said that that line of thought has lost acceptance and gave me some other material. This is a tricky issue for "high-level for c-suite".... Let me research further.